### ISSUE TITLE
On Firefox, JAWS doesn't read live regions when there is an aria-atomic="true" attribute.

### ISSUE THREAD

**Issue by juliemoynatPro** [FreedomScientific/VFO-standards-support#876]
### Summary

On Firefox, JAWS doesn't read live regions (with `aria-live="polite"` or `assertive` or with `role="status"`) when there is an `aria-atomic="true"` attribute. If I remove the attribute, the live region is read.

It works correctly with Chrome.

### Expected result

I expect JAWS reading the live region.

### Actual result

JAWS doesn't read live regions (with `aria-live="polite"` or `assertive` or with `role="status"`) when there is an `aria-atomic="true"` attribute.

### Example

It can be tested on [this CodePen example](https://codepen.io/copsae/pen/OJaEzzv).

#### JAWS version and build number

I've tested and it happens with JAWS 2025.24.10.106 and JAWS 2024.2409.2.
 
#### Operating System and version

Windows 11

#### Browser and version:

Firefox 133

**Comment by code-dispenser** [comment]
This issue has caused me some serious grief whilst trying to implement a stable cross-browser live region service.

Its still present in the latest version of JAWS when paired with FireFox and its well over a year since it was logged.

May I ask if you are planning to fix this or not so I can structure my code base accordingly? 

Thank you

Paul
